masterlist.yaml

```
@@ -14334,12 +14332,8 @@ plugins:
       - name: -Names
         condition: 'version("Guard Dialogue Overhaul.esp", "2.17", >=)'
     clean:
-      - crc: 0x6ABC1129
-        util: 'SSEEdit v4.0.2f'
-      - crc: 0x82FE6344
-        util: 'SSEEdit v4.0.2f'
-      - crc: 0x18AEFA4B
-        util: 'SSEEdit v4.0.3h (Hotfix 1)'
+      - crc: 0x21C5C3DB
+        util: 'SSEEdit v4.1.5f'
```

Two of those checksums are from the other version, so they should be kept.

```suggestion
      # Redux, True GDO Port
      - crc: 0x6ABC1129
        util: 'SSEEdit v4.0.2f'
      # Redux
      - crc: 0x82FE6344
        util: 'SSEEdit v4.0.2f'
      # SE
      - crc: 0x21C5C3DB
        util: 'SSEEdit v4.1.5f'
```